lib/compat/wordpress-6.2/rest-api.php

```diff
@@ -23,6 +23,15 @@ function gutenberg_register_rest_pattern_directory() {
 }
 add_action( 'rest_api_init', 'gutenberg_register_rest_pattern_directory' );
 
+/**
+ * Registers the block patterns REST API routes.
+ */
+function gutenberg_register_rest_block_patterns() {
+	$block_patterns = new Gutenberg_REST_Block_Patterns_Controller_6_2();
+	$block_patterns->register_routes();
+}
+add_action( 'rest_api_init', 'gutenberg_register_rest_block_patterns', 100 );
```

Comment:
What's the `priority=100` for?

Reply:
😅 I just copied from `6.1`. I'll remove.